pages/about.html

```diff
@@ -0,0 +1,129 @@
+                    gathering of
+                    cybersecurity experts and technology enthusiasts from around the world. This groundbreaking
+                    conference focuses on
+                    fortifying digital environments and safeguarding invaluable assets. Explore keynote presentations,
+                    interactive
+                    workshops, and thought-provoking sessions covering network security, threat intelligence, and more.
+                    Connect with
+                    professionals, leaders, and potential collaborators, fostering partnerships and exponential growth.
+                    Whether you're a
+                    cybersecurity aficionado, organizational luminary, or tech enthusiast, this summit offers a unique
+                    opportunity to expand
+                    knowledge and contribute to a resilient digital landscape. Join us on this extraordinary journey to
+                    shape the future of
+                    cybersecurity.
+                </div>
+                <div class="contact">
+                    <div class="p">Please contact us per Email for any further questions about <span class="bold">Otimus
+                            Cage Defender</span>
+                        Cybersecurity Summit 2023.
+                    </div>
+                    <span class="email">info@optimuscagedefender.org</span>
+
+                </div>
+            </div>
+        </section>
+
+        <section id="company-info">
+            <h3>
+                Optimus Cage Defender Logo and Company Information
+            </h3>
+            <p class="mobile-det">
+                The logo of Optimus Cage Defender is a shield with a cage in the middle. The cage is a metaphor for the
+                "Not personal info outlet" and The shield is protecting the cage from the outside world.
+            </p>
+
+            <p class="desktop-extra">
+                The logo competition, a celebration of creativity and innovation, concluded on May 20th this year. We
+                were thrilled
+                to witness the exceptional talent and ingenuity displayed by aspiring graphic designers. Amongst the
+                outstanding
+                entries, Peter Kibet's submission stood out, captivating hearts with its remarkable concept and
+                exquisite execution.
+                Peter's artistic prowess serves as an inspiration to young graphic designers, reminding them of the
+                limitless
+                possibilities that lie within their imagination. Embrace your unique vision, unleash your creativity,
+                and let your
+                designs paint a vibrant world of inspiration and storytelling. The future of graphic design rests in
+                your hands!
+            </p>
+
+
+            <div class="company-img">
+                <img src="./pageAssets/DefenderCage Logo.png" alt="">
+            </div>
```

Comment:
- [ ] I think it would be better to avoid using too many ```<div>``` tag in our codes, as suggested by the previous reviewer, you may simply remove the `<div>` tag and use only the `img` tag to make our codes more cleaner 🙏